### ISSUE TITLE
The run page might momentarily block during the auto-refresh.

### ISSUE THREAD

**Issue by rbf** [slipstream/SlipStreamUI#368]
We usually don't experience this if the SlipStream infrastructure is fast. As spotted by @konstan, on slow machines, the UI might become irresponsive for a couple secs every 15secs (the auto-refresh update).

Chrome already alerts from this behaviour:

```
jquery-1.11.1.min.js:4 Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.
```

**Comment by rbf** [comment]
Previous commit eb77fc6 should also fix #242.

**Comment by rbf** [comment]
Upstream dependency:

- [ ] slipstream/SlipStreamServer#422: Requesting a page without being authenticated redirects to `/login` page without returning a `401 Unauthorized` status code.
